src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -4828,6 +4829,8 @@ INSERT INTO tab_parameters VALUES('Z_MonoproticBaseMonoproticAcid','Z_Monoprotic
 INSERT INTO tab_parameters VALUES('Z_Neutral','Z_Neutral','Supporting parameter for calculation of account for drug ionisation (RR)','Dimensionless',NULL);
 INSERT INTO tab_parameters VALUES('Z_TriproticAcid','Z_TriproticAcid','Supporting parameter for calculation of account for drug ionisation (RR)','Dimensionless',NULL);
 INSERT INTO tab_parameters VALUES('Z_TriproticBase','Z_TriproticBase','Supporting parameter for calculation of account for drug ionisation (RR)','Dimensionless',NULL);
+INSERT INTO tab_parameters VALUES('Effective concentration in lumen','Effective concentration in lumen','In case of supersaturation: molecule amount divided by liquid volume. Otherwise: Min(intestinal solubility; molecule amount divided by liquid volume).','Concentration (molar)',NULL);

```

Comment:
This is a new parameter which is now used everywhere (transports, observers, ..)
With the formula `EnableSupersaturarion   ? M/Liquid : min(Solubility/MW ; M/Liquid)`

I wanted to introduce it since ever (even without supersaturation, because the formula was duplicated everywhere)